integration-tests/test/contracts.spec.ts

```diff
@@ -0,0 +1,202 @@
+import {BigNumber, Contract, ContractFactory, utils, Wallet} from 'ethers';
```

Comment:
I don't think this file is being linted

Reply:
Good catch. We have an error in our linting step. I'm going to make a PR to fix this.

Reply:
https://github.com/ethereum-optimism/optimism/pull/1697

Reply:
Fixed the lint issues manually.